content/posts/backlogged-pinball.md

```diff
@@ -0,0 +1,99 @@
++++
+title =  "Backlogged Pinball"
+tags = ["pinball", "gcp", "next"]
+date = "2024-10-07"
++++
+
+# Backlogged Pinball
+
+Earlier this year, I built a pinball-based demo for [Cloud Next 2024](https://cloud.withgoogle.com/next).  I'm preparing some posts about specific aspects of it, but I wanted to post a brief overview first.  Backlogged is a physical pinball machine that has custom software to connect it to Cloud
+services.
+
+
+{{< figure src="/images/backlogged-pinball/pinball_suit.jpeg" width="600px" title="A conference attendee playing the Backlogged pinball demo" >}}
+
+## Demo overview
+
+
+
+Backlogged pinball was designed to demonstrate specific cloud products and
```

Comment:
**Question:** Just double checking that you meant "cloud" (generic reference) — not "Cloud" (Google Cloud), right?

Reply:
Good point! I've updated references to 'Cloud' where appropriate.